webview-ui/src/hooks/useExtensionState.ts

```diff
@@ -17,6 +17,7 @@ const defaultState: ExtensionData = {
   workspaceRoot: "/",
   chatMessages: [],
   solutionState: "none",
+  solutionEffort: "Low",
```

Comment:
Could we pull in config value here?

Reply:
I would think this would cross a boundary we don't want to...right? Calling `vscode/.../utilities` from the webview-ui is what I thought I was avoiding.

Maybe I'm missing something and there is a better way to do this than defaulting to "Low" knowing it'll be overwritten when we actually activate the extension (from extension.ts):

```
    this.data = produce(
      {
        ...
        solutionEffort: getConfigSolutionMaxEffortLevel(),
      },
```

Reply:
Oh yeah that makes sense. Maybe we could just set it as undefined here then and only set it once we have the value.